website/src/components/CopySaveActions.svelte

```diff
@@ -0,0 +1,39 @@
+<script>
+  import Icon from './Icon/Icon.svelte'
+
+  let className = ''
+
+  export { className as class }
+  export let onSavedToClipboard
+  export let onSave
+
+  let isSavedToClipboard = false
+  let isSaved = false
+
+  const saveToClipboard = () => {
+    isSavedToClipboard = true
+    setTimeout(() => {
+      isSavedToClipboard = false
+    }, 700)
```

Comment:
Oh, and please move the `700` to a constant and name it 'saveToClipboardDelay' or similar.

Reply:
@jakubpawlowicz  I used the name `VALIDATION_ANIMATION_DURATION` because it's also used for the file download.